lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -539,6 +542,7 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
       ENV LIVEBOOK_TEAMS_NAME "#{socket.assigns.hub.hub_name}"
       ENV LIVEBOOK_TEAMS_OFFLINE_KEY "#{socket.assigns.hub.org_public_key}"
       ENV LIVEBOOK_TEAMS_SECRETS "#{encrypt_secrets_to_dockerfile(socket)}"
+      ENV LIVEBOOK_TEAMS_FS "#{encrypt_file_systems_to_dockerfile(socket)}"
```

Comment:
We should add this field (and LIVEBOOK_TEAMS_SECRETS) only if there is a secret/filesystem currently defined. :)